### DISCUSSION TITLE
Feature

### DISCUSSION THREAD

**Discussion by Temian1** [cjmellor/level-up#8] [Q&A]
Can user create two different points for different purposes and we can get the leaderboard depending on the type of points
For example
Point A ( for user activity)
Point B ( for purchasing goods)

**Comment by cjmellor** [comment]
I think -- if I understand correctly -- you could add the trait to a Model for User Activity and Purchases and add points that way.

Not sure about Leaderboard stuff, it's pretty basic at the minute.
